src/tests/ftest/util/file_count_test_base.py

```diff
@@ -95,6 +117,8 @@ def run_file_count(self):
                         self.ior_cmd.api.update('HDF5')
                         self.run_ior_with_pool(
                             create_pool=False, plugin_path=hdf5_plugin_path, mount_dir=mount_dir)
+                    elif self.ior_cmd.api.value == 'POSIX':
+                        self.run_ior_with_pool(create_pool=False, intercept=intercept)
```

Comment:
Similar - this is going to force all tests using this test base to use interception

Reply:
Yeah that's what we want as of now.